ui/pages/v2/KustomizationDetail.tsx

```diff
@@ -0,0 +1,114 @@
+import _ from "lodash";
+import * as React from "react";
+import styled from "styled-components";
+import Flex from "../../components/Flex";
+import KubeStatusIndicator from "../../components/KubeStatusIndicator";
+import Link from "../../components/Link";
+import Page from "../../components/Page";
+import ReconciledObjectsTable from "../../components/ReconciledObjectsTable";
+import Text from "../../components/Text";
+import { useGetKustomization } from "../../hooks/automations";
+import { AutomationKind } from "../../lib/api/core/types.pb";
+import { formatURL } from "../../lib/nav";
+import { V2Routes, WeGONamespace } from "../../lib/types";
+
+type Props = {
+  name: string;
+  className?: string;
+};
+
+const Info = styled.div`
+  padding-bottom: 32px;
+`;
+
+const InfoList = styled(
+  ({
+    items,
+    className,
+  }: {
+    className?: string;
+    items: { [key: string]: any };
+  }) => {
+    return (
+      <table className={className}>
+        <tbody>
+          {_.map(items, (v, k) => (
+            <tr key={k}>
+              <td>
+                <Text capitalize bold>
+                  {k}:
+                </Text>
+              </td>
+              <td>{v}</td>
+            </tr>
+          ))}
+        </tbody>
+      </table>
+    );
+  }
+)`
+  tbody tr td:first-child {
+    min-width: 100px;
+  }
+  tr {
+    height: 16px;
+  }
+`;
+
+function KustomizationDetail({ className, name }: Props) {
+  const { data, isLoading, error } = useGetKustomization(name);
+
+  const kustomization = data?.kustomization;
+
+  return (
+    <Page
+      title={kustomization?.name}
+      loading={isLoading}
+      error={error}
+      className={className}
+    >
+      <Info>
+        <h3>{kustomization?.namespace}</h3>
+        <InfoList
+          items={{
+            Source: (
```

Comment:
Do we not have the other fields that are in the Figma yet (namespace, lastUpdated, and interval)?

Reply:
Added!